src/filesystem/impls/appshell/AppshellFileSystem.js

```diff
@@ -536,19 +529,21 @@ define(function (require, exports, module) {
                 }
                 return;
             }
-            _nodeDomain.exec("watchPath", path)
+            _nodeDomain.exec("watchPath", path, ignored)
                 .then(callback, callback);
         });
     }
     /**
      * Stop providing change notifications for the file or directory at the
      * given path, calling back asynchronously with a possibly null FileSystemError
-     * string when the operation is complete.
+     * string when the operation is complete. This function needs to mirror the
+     * signature of watchPath and hence the ignored argument is passed from
+     * FileSystem code
```

Comment:
Could you put this on a new line? And maybe add one or max two line to say why is necessary?